lightning-invoice/src/utils.rs

```diff
@@ -510,6 +540,52 @@ where
 	}
 }
 
+
+/// Used to store information about all the HTLCs that are inflight across all payment attempts
+pub(crate) struct AccountForInFlightHtlcs<'a, S: Score> {
```

Comment:
Mh, do we come up with a name that mainly conveys what this object *is* in addition to what it *does*/its purpose is? Absolutely no blocker, but I find the naming a bit less descriptive than in could be. Unfortunately I'm not coming up with a good name either, just things like `InFlightHtlcScoreWrapper` or similar.

Reply:
Maybe S`corerAccountingForInFlightHtlcs`?

Reply:
resolved in [9606a8d](https://github.com/lightningdevkit/rust-lightning/pull/1694/commits/9606a8d52fbe6dfc2735c724128f56feb45ae76a)